ClassicsKotlin/README.md

```diff
@@ -17,8 +17,15 @@ Pre-requisites
 Getting Started
 ---------------
 
-This sample uses the Gradle build system. To build this project, use the
-"gradlew build" command or use "Import Project" in Android Studio.
+```sh
+git clone https://github.com/android/tv-samples.git
+```
+
+- Open the ClassicsKotlin project in [Android Studio][studio].
+- Compile and deploy to your Android TV device (such as a Nexus Player).
+
+Need more information about getting started with Android TV? Check the [official docs][getting-started].
```

Comment:
How about just linking directly e.g.

... the [official getting started guide](https://developer.android.com/training/tv/start/start.html)

Reply:
Looks like the link was right but I updated the text to "official getting started guide" across all the readme files to make it more clear.